app/src/main/res/layout/fragment_login.xml

```diff
@@ -42,14 +42,14 @@
             android:hint="@string/fragment_login_email"
             android:inputType="textEmailAddress"
             android:maxLines="1"
-            app:layout_constraintBottom_toTopOf="@id/favouriteColorInput"
+            app:layout_constraintBottom_toTopOf="@id/passwordInput"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="@+id/containerCover"
```

Comment:
I think here it should be `Top_toBottomOf="@+id/containerCover"`, because when the user inputs, the keyboard shouldn't lift up like it's shown in the images.

Reply:
If this doesn't fix the keyboard issue, try adding this:
`android:windowSoftInputMode="adjustPan"`
to your activity in your AndroidManifest